backend/coins/btc/account_test.go

```diff
@@ -189,6 +191,7 @@ func TestInsuredAccountAddresses(t *testing.T) {
 func TestSignAddress(t *testing.T) {
 	account := mockAccount(t, nil)
 	require.NoError(t, account.Initialize())
+	require.Eventually(t, func() bool { return account.Synced() }, time.Second, time.Millisecond*200)
```

Comment:
nit: isn't `func() bool { return account.Synced() }` the same as `account.Synced`?